staging/src/k8s.io/kubectl/pkg/cmd/config/create_cluster.go

```diff
@@ -108,8 +108,10 @@ func (o createClusterOptions) run() error {
 	if !exists {
 		startingStanza = clientcmdapi.NewCluster()
 	}
-	cluster := o.modifyCluster(*startingStanza)
-	config.Clusters[o.name] = &cluster
+	config.Clusters[o.name], err = o.modifyCluster(*startingStanza)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	cluster, err := o.modifyCluster(*startingStanza)
	if err != nil {
		return err
	}
	config.Clusters[o.name] = &cluster
```

Reply:
Made the change with a slight modification, because modifyCluster return type has changed from `clientcmdapi.Cluster` to `*clientcmdapi.Cluster` so that nil can be returned in case of an error.

```
config.Clusters[o.name] = cluster
```